apps/yarn.lock

```diff
@@ -60,7 +60,7 @@
     temporal latest
   optionalDependencies:
     browser-serialport latest
-    firmata "^0.12.0"
+    firmata latest
```

Comment:
On an initial pass at [0.11.3...0.16.0](https://github.com/firmata/firmata.js/compare/v0.11.3...v0.16.0) update in `playground-io` I had some board connection issues in my sensors test app.  I'll dig a little deeper and see if I can figure out what's wrong.  One wonders if `playground-io` should list `johnny-five` as a peerDependency since it's technically a johnny-five plugin.